imports/plugins/core/orders/server/no-meteor/resolvers/Order/payments.js

```diff
@@ -0,0 +1,34 @@
+import { xformOrderPayment } from "@reactioncommerce/reaction-graphql-xforms/order";
+
+/**
+ * @name Order/payments
+ * @method
+ * @memberof Order/GraphQL
+ * @summary Returns payments applied to an order
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} order order object refunds would be applied to
+ * @return {Promise<Object[]>} Promise that resolves with array of payment objects
+ */
+export default async function payments(context, order) {
+  if (Array.isArray(order.payments)) {
+    return order.payments.map(async (payment) => {
+      const xformPayment = xformOrderPayment(payment);
+
+      const refunds = await context.queries.refundsByPaymentId(context, {
```

Comment:
The reason I did this here, instead of inside the `xformOrderPayment` function, is I wanted to keep all the refunds xforms and data inside this orders package, instead of in the `core` package where `xformOrderPayment` is.

This could be moved into into  `xformOrderPayment` if that seems like a better place.